pgctl/cli.py

```diff
@@ -212,7 +219,7 @@ def unsupervise(self):
     def status(self):
         """Retrieve the PID and state of a service or group of services"""
         with self.pgdir.as_cwd():
-            for status in svstat(*self.services):
+            for status in svstat(*(service.alias for service in self.services)):
```

Comment:
Since you repeat this three times, it might make sense to grow a self.service_names property.